forms-test/src/main/java/org/odk/collect/formstest/InstanceUtils.kt

```diff
@@ -43,6 +43,6 @@ object InstanceUtils {
         val instanceDir = File(instancesDir + File.separator + System.currentTimeMillis() + Math.random())
         instanceDir.mkdir()
 
-        return createTempFile(instanceDir, "intance", ".xml")
+        return createTempFile(instanceDir, "intance_${System.currentTimeMillis()}", ".xml")
```

Comment:
What is this change for?

Reply:
Oh, it looks like a redundant change indeed. I needed this creating tests at some point I guess.